pkg/addons/adminconsole/adminconsole.go

```diff
@@ -40,12 +40,21 @@ var protectedFields = []string{
 	"automation",
 }
 
+const DEFAULT_ADMIN_CONSOLE_SVC_PORT = 30000
+
 var helmValues = map[string]interface{}{
 	"minimalRBAC":   false,
 	"isHelmManaged": false,
+	// To allow older versions of Admin Console to still work,
+	// we need to retain the "admin console service". It gets
+	// removed when kurlProxy is enabled.
```

Comment:
There are no older versions of admin console - we pin the version of the chart + kotsadm as part of the binary, so you can remove the "handle old versions" config items